moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -40,12 +50,37 @@ const FiltersChips = ({ filters, isOr, filtersConfig }) => {
     };
   });
 
+  const handleDeleteButtonClick = (filterIndex) => {
+    /* Clone the state, delete the filter index of the button clicked, and update filter state */
+    const filtersNewState = [...filters];
+    filtersNewState.splice(filterIndex, 1);
+    setFilters(filtersNewState);
+
+    if (filtersNewState.length > 0) {
+      /* If we have advanced filters, set query state values and update search params */
+      setSearchParams((prevSearchParams) => {
+        const jsonParamString = JSON.stringify(filtersNewState);
+
+        prevSearchParams.set(advancedSearchFilterParamName, jsonParamString);
+        return prevSearchParams;
+      });
+    } else {
+      /* Clear search params since we have no advanced filters */
+      setSearchParams((prevSearchParams) => {
+        prevSearchParams.delete(advancedSearchFilterParamName);
+        prevSearchParams.delete(advancedSearchIsOrParamName);
+
+        return prevSearchParams;
+      });
+    }
+  };
+
   return (
     <Box className={classes.filtersList}>
-      <Typography className={classes.filtersText}>
+      <Typography className={classes.filtersText} component="span">
         <Grid container alignItems={"center"} spacing={0.5}>
           {filtersCount > 1 && (
-            <Grid item spacing={0.25}>
+            <Grid item>
```

Comment:
`spacing` only works if its a container, removing it clears a warning in the console